doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -33,13 +33,12 @@ Traceability to Reference Model.
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `req.gen.ost.01` | Open source | The Architecture **must** use OpenStack APIs.|
 | `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through OpenStack APIs. |
-| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component.. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. |
 | `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
-| `req.gen.cnt.03` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where the state is required it **must** be kept external to the component. |
 | `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. |
```

Comment:
It should be explicitly mentioned if this is about the scaling of VNF-s or the scaling of the NFVI.

Reply:
@CsatariGergely req.lcm.gen.01 "The Architecture must support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement)." is in essence about NFVI scaling but we can clarify or add another requirement for NFVI scaling. Let us know. Thanks

Reply:
Oookay. What do you mean on policy driven and **auto** scaling in case of NFVI?